lib/configuration/variables/sources/instance-dependent/get-secretsmanager.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const ensureString = require('type/string/ensure');
+const ServerlessError = require('../../../../serverless-error');
+const _ = require('lodash');
```

Comment:
Use native versions instead of lodash

Reply:
I appreciate the review!  I looked around the code base a bit, and it doesn't appear very obvious to me where such a general purpose utility should live.  I placed it in the file, but please let me know if you spot a better space.